### PR TITLE
feat: Complete fizzbuzz

### PR DESCRIPTION
**Pull request by ujutech**
solve #1

### CODE REVIEW

File: fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+for i ini range(1,20):
```
Comment (keinetwork) [inline fizzbuzz.py:1]: it's not pythonic

File: fizzbuzz.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    if i%15==0:
```
Comment (keinetwork) [inline fizzbuzz.py:2]: Do fizzbuzz with 1 if statement

**Review by keinetwork — CHANGES_REQUESTED** [review]
DO this RIGHT AWAY !!!!!

File: fizzbuzz.py (hunk 1 above)
Comment (ujutech) [reply fizzbuzz.py:1]: Yes..

File: fizzbuzz.py (hunk 2 above)
Comment (ujutech) [reply fizzbuzz.py:2]: I'll do it

**Review by keinetwork — APPROVED** [review]
아니 .. 할줄 아시면서 지금..

**Pull request merged**